sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -184,6 +186,29 @@ public void assign(final List<TopicPartition> assigned) {
               updateEndOffsets(ImmutableMap.of(tp, (long) records.get(tp).size()));
             }
           }
+          public Map offsetsForTimes(Map<TopicPartition, Long> timestampsToSearch) {
```

Comment:
For working with kafka 0.9.
We should return Map<TopicPartition, OffsetAndTimestamp>, but 0.10.1.0 previous versions does not have the OffsetAndTimestamp class. So we return a raw type and use reflection here.